src/command-handlers/exec.ts

```diff
@@ -0,0 +1,35 @@
+import { Command } from 'commander';
+import winston from 'winston';
+import { spawn } from 'child_process';
+import { getVaultSecret, initVaultSecrets } from '../modules/database';
+
+export const runExec = async (_options: unknown, program: Command) => {
+    const command = program.args.join(' ');
+    const environmentVariables = process.env;
+
+    await initVaultSecrets();
+
+    for (const key in environmentVariables) {
```

Comment:
Isn't simplet to do `for (const [key, value] of Object.entries(process.env)) {console.log({key, value})}` ?

Reply:
absolutely 😓